util/messages.py

```diff
@@ -1,10 +1,8 @@
-import json
 import time
 
 from datetime import datetime
 
-MESSAGES_FILES = "resources/user-messages/messages.json"
-SLEEP_TIME = 3
+from settings import SLEEP_TIME
 
 
```

Comment:
Put this functions inside a class and put them into services, this must not be in utils.